sdk/cosmos/azure-cosmos-benchmark/src/main/java/com/azure/cosmos/benchmark/linkedin/GetTestRunner.java

```diff
@@ -0,0 +1,158 @@
+            int index = (int) ((i % keys.size()) % Integer.MAX_VALUE);
+            final Key key = keys.get(index);
+            _executorService.submit(() -> runOperation(key, testData, errorCount));
+        }
+
+        try {
+            _executorService.awaitTermination(10, TimeUnit.SECONDS);
+        } catch (InterruptedException e) {
+            LOGGER.error("Error awaiting the completion of all tasks", e);
+        }
+
+        final Instant runEndTime = Instant.now();
+        LOGGER.info("Number of iterations: {}, Errors: {}, Runtime: {} millis", i,
+            errorCount.get(),
+            runEndTime.minusMillis(runStartTime).toEpochMilli());
+        _executorService.shutdown();
+    }
+
+    private void runOperation(final Key key, final Map<Key, ObjectNode> testData, final AtomicLong errorCount) {
+        try {
+            final Result<Key, JsonNode> result = _accessor.get(key, GetRequestOptions.EMPTY_REQUEST_OPTIONS);
+            if (VALIDATE_RESULTS && !expectedResponse(testData.get(key), result)) {
+                LOGGER.info("Result mismatch for Key {}; Actual value: {}, Expected: {}", key, result.getResult(),
+                    testData.get(key));
+                errorCount.getAndIncrement();
+            }
+        } catch (AccessorException e) {
+            errorCount.getAndIncrement();
+        }
+    }
+
+    private boolean expectedResponse(final ObjectNode expectedResult, final Result<Key, JsonNode> result) {
+        final JsonNode actualValue = result.getResult()
+            .map(Entity::get)
+            .orElse(null);
+        final Iterator<String> fieldNames = expectedResult.fieldNames();
+        while (fieldNames.hasNext()) {
+            final String field = fieldNames.next();
+            final JsonNode expectedJsonNode = expectedResult.get(field);
+            final JsonNode actualJsonNode = actualValue.get(field);
+            if (!expectedJsonNode.equals(actualJsonNode)) {
+                LOGGER.info("mismatch Actual value: {}, Expected: {}", actualJsonNode, expectedJsonNode);
+                return false;
+            }
+        }
+        return true;
+    }
+
+    private Accessor<Key, JsonNode> createAccessor(final Configuration configuration,
+        final CosmosAsyncClient client,
+        final MetricRegistry metricsRegistry) {
+
+        final StaticDataLocator dataLocator = createDataLocator(configuration, client);
+        final KeyExtractor<Key> keyExtractor = new KeyExtractorImpl();
+        final DocumentTransformer<JsonNode, JsonNode> documentTransformer = new IdentityDocumentTransformer<>();
+        final Clock clock = Clock.systemUTC();
+        return new CosmosDBDataAccessor<>(dataLocator,
+            keyExtractor,
+            new ResponseHandler<>(documentTransformer, keyExtractor),
+            new MetricsFactory(metricsRegistry, clock),
+            clock,
+            new OperationsLogger(Duration.ofMillis(100)));
```

Comment:
Can we make this value configurable

Reply:
See my comment above. I'd rather set it to a high value than introduce another param in Configuration for this niche scenario.